sdk/go/asymmetricdecryopt.go

```diff
@@ -0,0 +1,87 @@
+package ehsm
+
```

Comment:
file name reference the python folder, e.g
asymmetric_decrypt.go

Reply:
fix